src/sidebar/components/test/annotation-header-test.js

```diff
@@ -93,5 +93,31 @@ describe('sidebar.components.annotation-header', function () {
         assert.deepEqual(ctrl.displayName(), 'Bill Jones');
       });
     });
+
+    describe('#thirdPartyUsernameLink', () => {
+      it('returns the thirdPartyUsernameLink if set', () => {
+        var ann;
+        var ctrl;
+
+        fakeSettings.usernameUrl = 'http://www.example.org/';
+        ann = fixtures.defaultAnnotation();
+        ctrl = $componentController('annotationHeader', {}, {
+          annotation: ann,
+        });
+        assert.deepEqual(ctrl.thirdPartyUsernameLink(), 'http://www.example.org/bill');
+      });
+
+      it('returns null if thirdPartyUsernameLink not set in the settings object', () => {
```

Comment:
Re. the description of this test, In the settings object the property is called "usernameLink" rather than "thirdPartyUsernameLink".